### PR TITLE
Fix index layout

### PR DESCRIPTION
**Pull request by 4RN4LDO**
-Fix layout on index
-Add style on back|edit buttons

### CODE REVIEW

File: app/views/partials/_search_bar.html.haml [hunk 1]
```diff
@@ -2,11 +2,13 @@
   .filters-bar
     .navbar
       .navbar-collapse.collapse
-        %ul.nav.navbar-nav
-          %li
+        .navbar-form
```
Comment (jramoscr) [inline app/views/partials/_search_bar.html.haml:5]: This only centers the text? Why don't you use the `text-center` class from bootstrap?

File: app/views/musicians/show.html.haml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-= render partial: 'partials/cover_search'
+= render partial: 'partials/cover_board', locals: { resource: @musician}
```
Comment (jramoscr) [inline app/views/musicians/show.html.haml]: `... resource: @musician }`. Here you missed the space.

File: app/views/musicians/show.html.haml [hunk 3]
```diff
@@ -59,6 +59,9 @@
 / %p Email: #{ @musician.user.email }
 / %p Telefono: #{ @musician.phone }
 
-= link_to 'Edit', edit_musician_path(@musician)
-|
-= link_to 'Back', musicians_path
+%nav
+  %ul.pager
+    %li
+      = link_to 'Edit', edit_musician_path(@musician)
```
Comment (jramoscr) [inline app/views/musicians/show.html.haml:65]: This string should be in spanish, right? `Edit` -> `Editar`

File: app/views/musicians/show.html.haml [hunk 4]
```diff
@@ -59,6 +59,9 @@
...
+    %li
+      = link_to 'Back', musicians_path
```
Comment (jramoscr) [inline app/views/musicians/show.html.haml:67]: Same here.

File: app/views/partials/_search_bar.html.haml (hunk 1 above)
Comment (aleks279) [inline app/views/partials/_search_bar.html.haml:5]: CSS selector.

That `div` is containing the form created by the `form_tag` I placed later. `text-center` as the name implies is for centering text only.

**Comment by aleks279** [comment]
Need this for demo. Merging.

**Pull request merged**